src/collections.js

```diff
@@ -0,0 +1,86 @@
+import {
+  actionToPromise,
+} from 'core';
+
+/** Create an action that runs several actions at once and resolves as soon
+ * as any one of the actions resolves.
```

Comment:
for jsdoc comments, the first line shouldn't have any text, just the opening comment with an extra asterisk
```suggestion
/** 
 * Create an action that runs several actions at once and resolves as soon
 * as any one of the actions resolves.
```